docs/releases/upgrading.rst

```diff
@@ -0,0 +1,110 @@
+| 0.4               | 1.6                          | 2.6, 2.7, 3.2, 3.3, 3.4     | 
++-------------------+------------------------------+-----------------------------+ 
+| 0.5               | 1.6                          | 2.6, 2.7, 3.2, 3.3, 3.4     | 
++-------------------+------------------------------+-----------------------------+ 
+| 0.6               | 1.6, 1.7                     | 2.6, 2.7, 3.2, 3.3, 3.4     | 
++-------------------+------------------------------+-----------------------------+ 
+| 0.7               | 1.6, 1.7                     | 2.6, 2.7, 3.2, 3.3, 3.4     | 
++-------------------+------------------------------+-----------------------------+ 
+| 0.8 LTS           | 1.6, 1.7                     | 2.6, 2.7, 3.2, 3.3, 3.4     | 
++-------------------+------------------------------+-----------------------------+ 
+| 1.0               | 1.7, 1.8                     | 2.7, 3.3, 3.4               | 
++-------------------+------------------------------+-----------------------------+ 
+| 1.1               | 1.7, 1.8                     | 2.7, 3.3, 3.4               | 
++-------------------+------------------------------+-----------------------------+ 
+| 1.2               | 1.7, 1.8                     | 2.7, 3.3, 3.4, 3.5          | 
++-------------------+------------------------------+-----------------------------+ 
+| 1.3               | 1.7, 1.8, 1.9                | 2.7, 3.3, 3.4, 3.5          | 
++-------------------+------------------------------+-----------------------------+ 
+| 1.4 LTS           | 1.8, 1.9                     | 2.7, 3.3, 3.4, 3.5          | 
++-------------------+------------------------------+-----------------------------+ 
+| 1.5               | 1.8, 1.9                     | 2.7, 3.3, 3.4, 3.5          | 
++-------------------+------------------------------+-----------------------------+ 
+| 1.6               | 1.8, 1.9, 1.10               | 2.7, 3.3, 3.4, 3.5          | 
++-------------------+------------------------------+-----------------------------+ 
+| 1.7               | 1.8, 1.9, 1.10               | 2.7, 3.3, 3.4, 3.5          | 
++-------------------+------------------------------+-----------------------------+ 
+| 1.8 LTS           | 1.8, 1.9, 1.10               | 2.7, 3.3, 3.4, 3.5          | 
++-------------------+------------------------------+-----------------------------+ 
+| 1.9               | 1.8, 1.9, 1.10               | 2.7, 3.3, 3.4, 3.5          | 
++-------------------+------------------------------+-----------------------------+ 
+| 1.10              | 1.8, 1.10, 1.11              | 2.7, 3.4, 3.5, 3.6          | 
++-------------------+------------------------------+-----------------------------+ 
+| 1.11              | 1.8, 1.10, 1.11              | 2.7, 3.4, 3.5, 3.6          | 
++-------------------+------------------------------+-----------------------------+ 
+| 1.12 LTS          | 1.8, 1.10, 1.11              | 2.7, 3.4, 3.5, 3.6          | 
```

Comment:
As far as mentioning LTS support, are we going to have End-of-Life dates as well?